docs/source/specification.md

```diff
@@ -72,3 +78,35 @@ A real project folder may look like:
             └── histology/
                 └── cell_counts.csv
 ```
+
+## File Naming Conventions
+
+File names are *required* to be formatted as set of key-value pairs ending with an *optional* suffix and *required* file extension (e.g. `ses-001_date-20220516_retinotopy.ap.bin`). 
+
+Key-value pairs are separated by underscores while the key and values 
+separated by hypens (e.g. `sub-001_key1-value1_key2-value2.csv`). Key-value pairs other than the *required* `sub` or `ses` are *optional* and can contain 
```

Comment:
Did we decide that `sub` and `ses` are also required in the filename? I thought everything in filenames is convention now (except for requiring an extension)

Reply:
Yeah I don't think we can require anything from filenames as they are sometimes generated automatically by other software.

Reply:
We can certainly suggest a naming strategy though.